src/_gettsim/transfers/kinderbonus/kinderbonus.py

```diff
@@ -1,12 +1,18 @@
-def kinderbonus_m(kindergeld_m: float, kindergeld_params: dict) -> float:
+"""Kinderbonus."""
+
+from _gettsim.functions.policy_function import policy_function
+
+
+@policy_function()
```

Comment:
So we always have this and set the parameters to zero if it is not active? Might be nicer to work with `dates_active`?